### ISSUE TITLE
Gradient instead of series of colors for audio visualizer effect

### ISSUE THREAD

**Issue by milovincent** [rgb-master-team/RGBMaster#45]
would be nice to have a gradient setup for said effect instead of setting every color individually

**Comment by deankevorkian** [comment]
Hey, sorry for the delayed answer, been having some trouble working on the project lately. Can you elaborate? I'm not sure what you mean exactly

**Comment by milovincent** [comment]
not sure how much more clear I can be. I want to be able to dictate a gradient that the colors move along as opposed to setting 100 different colors bc I want some different color scheme than the default.

**Comment by milovincent** [comment]
sorry, don't want to be unhelpful, what exactly are you confused about?

**Comment by deankevorkian** [comment]
You cleared it up for me. To make sure I understand, you want to set a color for every volume point, but have the colors gradient between them instead of immediately change, right?

**Comment by milovincent** [comment]
no, ideally I'd like the volume points to be interpolated along a smooth gradient defined by 2 or more colors I do set.

**Comment by MrKlorox** [comment]
also preferably with a way to choose how it fades to the next color. be it direct or counter/clockwise hue rotation.
